### PR TITLE
add service menu and some fixes

### PR DESCRIPTION
**Pull request by zadziarnouksi-taras**

### CODE REVIEW

File: coffeeMachineToShow.cpp [hunk 1]
```diff
@@ -64,22 +66,49 @@ int main() {
...
+                    flagService = true;
+                    while (flagService) {
+                        printServiceMenu();
+
+                        choice = inputNumber(choice);
+
+                        if (choice == 1) {  //Money
+                            printBalance(balanceInMachine, "money");
+
+                            pause();
```
Comment (dzmitri-litvinov) [inline coffeeMachineToShow.cpp:82]: IAs for me, it's better to use  `printBalance(balance + balanceInMachine, "money");` which displays the total amount of money in the machine.
What do you think?

File: coffeeMachineToShow.cpp [hunk 2]
```diff
@@ -64,22 +66,49 @@ int main() {
...
+                        } else if (choice == 2) {   //Cups
+                            cups = inputMaterials(cups, "cups");  //max 700 pcs
+                        } else if (choice == 3) {   //Withdraw money
+                            transaction(balanceInMachine, balanceInMachine, moneyTakenOut);
+                            transaction(balance, balance, moneyTakenOut);
+
+                            cout << "Operation completed successfully !!!" << endl;
```
Comment (dzmitri-litvinov) [inline coffeeMachineToShow.cpp:89]: Maybe we can add something like `cout << "Operation completed successfully !!! Money withdrawn: " << moneyTakenOut << endl;` ?

**Review by dzmitri-litvinov — COMMENTED** [review]
Please find my comments below

File: coffeeMachineToShow.cpp (hunk 1 above)
Comment (zadziarnouksi-taras) [reply coffeeMachineToShow.cpp:82]: Agree with you. Good job!

File: coffeeMachineToShow.cpp (hunk 2 above)
Comment (zadziarnouksi-taras) [reply coffeeMachineToShow.cpp:89]: Good idea!

**Pull request merged**